view-user/src/components/common/BingoResult/BingoResult.tsx

```diff
@@ -1,36 +1,90 @@
 import React from "react";
 import Image from "next/image";
 import styles from "./BingoResult.module.css";
-import { BingoIcon } from "@/components/common";
+import { BingoIcon, Button } from "@/components/common";
 import { BingoNumber } from "@/utils/api_methods";
+import { useState } from "react";
 
 interface BingoResultProps {
   bingoResultNumber: BingoNumber[];
 }
 
 export const BingoResult = (props: BingoResultProps) => {
+  const [resultChange, setResultChange] = useState<boolean>(true);
   const copiedArray = [...props.bingoResultNumber];
+  const sortCopiedArray = [...props.bingoResultNumber];
   const firstBingoNumber = copiedArray.pop();
+  const sortFirstBingoNumber = sortCopiedArray
+    .sort((a, b) => a.data - b.data)
+    .shift();
+
+  function ResultNumber() {
+    if (resultChange) {
+      return (
+        <>
+          <div className={styles.card_frame}>
+            <div className={styles.large_card}>{firstBingoNumber?.data}</div>
+            <div className={styles.small_card_frame}>
+              {[...props.bingoResultNumber]
+                .slice(0, -1)
+                .reverse()
+                .map((num, index) => (
+                  <div className={styles.small_card} key={index}>
+                    <div>{num.data}</div>
+                  </div>
+                ))}
+            </div>
+          </div>
+        </>
+      );
+    } else {
+      return (
+        <>
+          <div className={styles.card_frame}>
+            <div className={styles.large_card}>
+              {sortFirstBingoNumber?.data}
+            </div>
+            <div className={styles.small_card_frame}>
+              {[...props.bingoResultNumber]
+                .sort((a, b) => a.data - b.data)
+                .slice(1)
+                .map((num, index) => (
+                  <div className={styles.small_card} key={index}>
+                    <div>{num.data}</div>
+                  </div>
+                ))}
+            </div>
+          </div>
+        </>
+      );
+    }
+  }
 
   return (
     <div className={styles.content_wrapper}>
       <div className={styles.container}>
         <div className={styles.frame_title}>
-          <Image src="/BingoCard.svg" alt="BingoCard" width={25} height={25} />
+          <Image src="/BingoCard.svg" alt="BingoCard" width={20} height={20} />
           <p>BINGO Number</p>
+          <div className={styles.frame_title_button}>
+            <Button
+              size="s"
+              shape="square"
+              onClick={() => setResultChange(true)}
+            >
+              抽選順
+            </Button>
+            <Button
+              size="s"
+              shape="square"
+              onClick={() => setResultChange(false)}
+            >
+              昇　順
+            </Button>
+          </div>
```

Comment:
押しやすくするため、ボタンを下部に配置したほうが良いかもしれません。
また「昇　順」の文字間の空白が全角スペースになっていたため、半角スペースにするか、
「番号順」のような「抽選順」と同じ3文字の漢字の方が見た目が整うかと思いました。


（押しやすくするためのボタンサイズとして「.xm」を追加しました。）
```css
.xm {
  font-size: x-small;
  padding: 0.5rem 2rem;
}

``` 


![FireShot Capture 082 - Bingo - localhost](https://github.com/NUTFes/nutfes-Bingo/assets/131145590/5888a839-2583-4b6c-8722-c9c22657fcc0)

```suggestion
         </div>
        <div className={styles.frame_title_button}>
          <Button
            size="xm"
            shape="square"
            onClick={() => setResultChange(true)}
          >
            抽選順
          </Button>
          <Button
            size="xm"
            shape="square"
            onClick={() => setResultChange(false)}
          >
            番号順
          </Button>
```

Reply:
下に配置するの私的にはそんなに好きじゃなかったので横に置いてました。
下部の方がいいですかね？

Reply:
配置は好みだと思います！
現状だとボタンサイズが小さく、２つのボタンの押し分けが難しく感じました。
横に配置するなら大きめのボタン１つで交互の切り替えにしてみるとかどうでしょう？
( 参考：https://note.com/gunj/n/n165a2f8ba2ad )